server/GRPC_TA2_TA3/core.proto

```diff
@@ -0,0 +1,235 @@
+syntax = "proto3";
```

Comment:
Please remove files from the API proto repo. We will actually sync them with gitlab separately. For example, you can assume the files reside in GRPS_TA2_TA3/ta2ta3-api/.